app/src/main/java/com/tip/lunchbox/model/server/request/FavouriteRestaurants.java

```diff
@@ -0,0 +1,16 @@
+package com.tip.lunchbox.model.server.request;
+
+import com.google.gson.annotations.Expose;
+
+public class FavouriteRestaurants {
+    @Expose
+    private String zomatoResId;
```

Comment:
```suggestion
    @SerializedName("zomato_res_id")
    @Expose
    private String zomatoResId;
```

Also, this should be an Integer value, the API does not accept String.

Reply:
@epicadk check this, the datatype (as I tested in postman) expected is integer.